### PR TITLE
Added px-backup upgrade related env variables

### PR DESCRIPTION
**Pull request by ak-px**
**What this PR does / why we need it**:
Added px-backup upgrade related env varibales LATEST_PXBACKUP_VERSION & UPGRADE_STORK_IMAGE which is required for upgrade pipelines
**Which issue(s) this PR fixes** (optional)
Closes #PA-1002

### CODE REVIEW

**Review by mkoppal-px — APPROVED** [review]
LGTM except for a minor comment

File: deployments/deploy-ssh.sh [hunk 1]
```diff
@@ -634,6 +634,10 @@ spec:
       value: "${LOGGLY_API_TOKEN}"
     - name: PODMETRIC_METERING_INTERVAL_MINUTES
       value: "${PODMETRIC_METERING_INTERVAL_MINUTES}"
+    - name: LATEST_PXBACKUP_VERSION
+      value: "${LATEST_PXBACKUP_VERSION}"
+    - name: UPGRADE_STORK_IMAGE
+      value: "${UPGRADE_STORK_IMAGE}"
```
[the remark this answers is not in the document]
Comment (ak-px) [reply deployments/deploy-ssh.sh]: Updated it . Made changes to backup_helper.go as well.

**Review by vpinisetti-px — APPROVED** [review]
Optional Comment. I believe, it's good to have generic variable names something like below. In that  way it refers target version whether its latest or any hotfix version etc.
TARGET_PXBACKUP_VERSION
TARGET_STORK_VERSION

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**